RELEASE.md

```diff
@@ -10,6 +10,4 @@
 - Once the above PR is merged, wait until an automatic PR is opened against stable which updates the checksums.
 - Once the above PR is merged, tag the release `v{version-here}` and push the tag.
 - Wait until an automatic PR is opened against the develop branch updating the checksums in `README.rst`. Merge it.
-- Check that an automated PR was opened against the salt repo updating the bootstrap script, located in `salt/cloud/deploy/bootstrap-salt.sh`
```

Comment:
I think we still need to know that we need to update that file, otherwise it will be forgotten

Reply:
Updated! Ready for re-review